static/compiler.js

```diff
@@ -601,6 +601,7 @@ Compiler.prototype.onCompileResponse = function (request, result, cached) {
 
     this.compileTimeLabel.text(timeLabelText);
 
+    result.stderr = result.stderr.slice(0, options.truncateErrorMessages);
```

Comment:
The PR says lines, but I had thought stderr was a buffer so won't this just be taking 50 chars?

Reply:
Things appear to work as expected with Rust as well.

Reply:
Great. 

Confused even the node docs seem to indicate it should just be a string. But if it works, it works.